pkg/backup/s3/s3.go

```diff
@@ -114,8 +116,27 @@ func (s *S3) List() ([]string, error) {
 	keys := []string{}
 	for _, key := range resp.Contents {
 		k := *key.Key
-		keys = append(keys, k[len(s.prefix):])
+		keys = append(keys, k[len(prefix):])
 	}
 
 	return keys, nil
 }
+
+func (s *S3) CopyDir(from string) error {
```

Comment:
Logically, we are copying a dir. Because S3 doesn't have dir, that's why we need to implement this method :(